src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -450,7 +445,7 @@ public void TestMainModule()
                 (Func<string, string>)((s) => s.ToLowerInvariant()) :
                 (s) => s;
 
-            Assert.True(p.Modules.Count > 0);
+            Assert.InRange(p.Modules.Count, 1, int.MaxValue);
```

Comment:
I'd change Max to something high but reasonable (5000?) to detect stupid errors

Reply:
Ditto